### ISSUE TITLE
Add image to chapter

### ISSUE THREAD

**Issue by henri9813** [BookStackApp/BookStack#1783]
**Describe the feature you'd like**
I would like to have the possibility to add a image cover to a chapter

**Describe the benefits this feature would bring to BookStack users**
Add image on a chapter

**Additional context**
It's only for  design purpose, i really like the idea to have an image as cover of chapter ( like the book's cover )

Thank you :)

**Comment by franbarone** [comment]
I would also like to be able to add an image at the chapter level.

**Comment by SonGokussj4** [comment]
Yeah. Right now the chapters are not well distinguished. Optional image would help massively!

**Comment by nheimsath-gudat** [comment]
Optional Image would be extremely helpfull. Alternatively it would be cool to have Font Awesome Icons available for chapters.

**Comment by muellerryan** [comment]
This would be super useful since it's difficult to quickly select a chapter I want to read until I read the titles of each one.

**Comment by nheimsath-gudat** [comment]
That would help alot !!!

**Comment by chrisweeksnz** [comment]
Both for technical and fiction content, chapter images would be great!

**Comment by ssddanbrown** [comment]
Additional request for this from the BookStack support service (Ticket 62).

> it would be really nice if there was a possibility to add and image to each chapter of the book (and not only in pages)
> The image would be at the bottom of course.
> It would make the book a bit more colorful and interesting…

**Comment by jaretclifton** [comment]
+1 for this!

**Comment by balta1980** [comment]
I agree this. Nice!!

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/1783
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
